frontend/src/css/PublicProfile.styl

```diff
@@ -34,6 +48,8 @@
     padding-right pageSpacing
     background-color offWhite
     display flex
+    &__callBtn
+      margin 6px
```

Comment:
Why is this not using a standard margin?

Reply:
I will fix this in another ticket for improvement, I used this to resize the call button